src/test/java/ch/uzh/ifi/hase/soprafs22/service/GameServiceTest.java

```diff
@@ -376,4 +376,35 @@ void getGameFromRandomUser_throwNotFound() {
         ResponseStatusException exception = assertThrows(ResponseStatusException.class, () -> gameService.getGameFromRandomUser(testUser));
         assertEquals("404 NOT_FOUND \"There is no black card of another user left\"", exception.getMessage());
     }
+
+    @Test
+    void haversineDistance_correctOutput() {
+
+        double lat1 = 0;
+        double lon1 = 0;
+        double lat2 = 10;
+        double lon2 = 10;
+        
+        double distance = GameService.haversineDistance(lat1, lon1, lat2, lon2);
+
+        // test if result is the same when rounded to 2 decimal places.
+        // actual result computed using: https://www.vcalc.com/wiki/vCalc/Haversine+-+Distance (not the same source as for the code)
+        assertEquals(distance, 1568.52, 0.01);
```

Comment:
in assertEquals, in most programming languages (if not all nowadays) it is convention to first have expected as first argument, and actual as second argument. Can you change that, so that it matches with the rest of our tests?

Reply:
fair point